WebRoot/css/zm.css

```diff
@@ -6616,3 +6604,159 @@ a.ZmLinkDisabled {
 .ZmContactInfoView .ZmContactGroupListContent .ZmContactGroupItemTable .ZmContactGroupItemImage {
 	@ContactGroupItemViewImageWrapper@
 }
+
+/* Contact Group Create Edit */
+.ZmContactView {
+	@GroupEditView@
+}
+
+.ZmContactView .ZmGroupViewWrapper {
+	@GroupEditViewWrapper@
+}
+
+.ZmContactView .ZmGroupViewWrapper input[type='text'],
+.ZmContactView .ZmGroupViewWrapper textarea {
+	@GroupEditViewFontSize@
+}
+
+.ZmGroupViewWrapper .ZmContactHeaderTags {
+	@GroupEditTagsWrapper@
+}
+
+.ZmGroupViewWrapper .ZmContactHeaderLocation {
+	@GroupEditLocation@
+}
+
+.ZmGroupViewWrapper .ZmContactHeaderLocationLabel {
+	@GroupEditLocationLabel@
+}
+
+.ZmGroupViewWrapper .ZmContactHeaderLocationName .ZButtonBorder {
+	@GroupEditLocationNameWrapper@
+}
+
+.ZmGroupViewWrapper .ZmContactHeaderLocationName .ZWidgetTitle {
+	@GroupEditLocationName@
+}
+
+.ZmGroupViewWrapper .ZmContactHeaderLocationName .ZWidgetIcon {
+	@GroupEditLocationIcon@
+}
+
+.ZmGroupViewWrapper .ZmContactHeaderLocationName :not(.ZDisabled) .ZButtonBorder:hover .ZWidgetTitle,
+.ZmGroupViewWrapper .ZmContactHeaderLocationName :not(.ZDisabled) .ZButtonBorder:focus .ZWidgetTitle,
+.ZmGroupViewWrapper .ZmContactHeaderLocationName :not(.ZDisabled) .ZButtonBorder:active .ZWidgetTitle {
+	@GroupEditLocationNameActive@
+}
+
+.ZmGroupViewWrapper .ZmGroupMembersWrapper {
+	@GroupMembersWrapper@
+}
+
+.ZmGroupViewWrapper .ZmContactContent {
+	@GroupEditViewContent@
+}
+
+.ZmEditGroupContact {
+	@GroupEditContentListWrapper@
+}
+
+.ZmEditGroupContact .Row {
+	@GroupEditContentListRow@
+}
+
+.ZmEditGroupContact .Row.Row-selected {
+	@GroupEditContentListRowSelected@
+}
+
+.ZmEditGroupContact .Row TD,
+.ZmEditGroupContact .Row DIV {
+	@GroupEditContentListRowText@
+}
+
+.ZmEditGroupContact .Row.Row-selected TD,
+.ZmEditGroupContact .Row.Row-selected DIV {
+	@GroupEditContentListRowSelectedText@
+}
+
+.ZmEditGroupContact .ZmContactGroupItemTable {
+	@GroupEditContentItemTable@
+}
+
+.ZmGroupAddMemberBtnWrapper {
+	@GroupAddMemberBtnWrapper@
+}
+
+.ZmGroupAddMemberBtn .ZButton {
+	@GroupAddMemberBtn@
+}
+
+.ZmGroupAddMemberBtn .ZButton:hover,
+.ZmGroupAddMemberBtn .ZButton:focus,
+.ZmGroupAddMemberBtn .ZButton:active,
```

Comment:
do we need to use both selectors..the pseudo selectors and equivalent class selectors . 
:hover & ZHover for example